content/includes/header.html

```diff
@@ -34,6 +34,9 @@
 <meta content="{{ tag }}" property="article:tag">
 {% endfor %}
 {% endif %}
+{% if title == 'Education Benefits Application Process' or title == 'Education Benefits Eligibility' %}
+<link rel="import" href="/education/wizard/index.html">
```

Comment:
I'm not entirely sure how that relates to `include` (that issue is also for Jekyll). What you're doing looks really similar to the way the sticky action box worked. It looks like you want to be able to include some common html on two different pages, which I think you can do just by putting `{% include ... }` on each of those pages, in the right spot.